.gitignore

```diff
@@ -9,6 +9,7 @@ node_modules
 .env.development.local
 .env.test.local
 .env.production.local
+.envrc
```

Comment:
I use a local `.envrc` file sometimes to test connecting a locally-built container to a remote DB URL.